pkg/wireguard-tools/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM linuxkit/alpine:3744607156e6b67e3e7d083b15be9e7722215e73 as tools
+RUN echo http://dl-cdn.alpinelinux.org/alpine/edge/main > /etc/apk/repositories && \
+    echo http://dl-cdn.alpinelinux.org/alpine/edge/testing >> /etc/apk/repositories && \
+    apk update && \
```

Comment:
I dont think there is any point having a wireguard-tools package, it doesnt actually do anything - my suggestion was adding these to tools/alpine directly so they can easily be added to another package, and then make packages that do something specific, like setting up a vpn.